### ISSUE TITLE
Example for animation?

### ISSUE THREAD

**Issue by jimver04** [microsoft/glTF-SDK#115]
Hi,
The provided samples are very limited . 
is there a serialization example for exporting animated meshes in glb? Rough directions?
Thanks, 
Dimitrios

**Comment by bghgary** [comment]
Maybe try looking here: https://github.com/BabylonJS/BabylonPolymorph/tree/master/Morphs/Importers/Transcoders/Transcoders/Source

**Comment by jimver04** [comment]
Hi, 

the BabylonPolymorph was really a help. 
I have achieved animation for the morph targets by replacing the VerticesArray per each step: 

```
accessorIdPositionsTargets[stepidx] = bufferBuilder.AddAccessor(VerticesArray, accessorDescMorphingCoordinates).id;
meshPrimitive.targets[stepidx].positionsAccessorId = accessorIdPositionsTargets[stepidx];
```

Can I animate also Vertex colors ? From I what I see I can not put them in the TargetPath of the morph. 

![image](https://user-images.githubusercontent.com/1898502/209144556-c8690fa8-4c26-430a-baae-a490cd3d768f.png)

Best,
Dimitrios

**Comment by bghgary** [comment]
Not with the core spec. There is this extension that animates other properties, but I still don't think you can animate vertex colors. That will be difficult to implement since vertex attributes will have to be updated every frame.

**Comment by SubDrag** [comment]
I agree - the examples need a ton of work. The Serialize/Deserialize should really fan through almost all parts of all the different type in a normal parsing way.  They're too limited.
